lib/internal/crypto/webidl.js

```diff
@@ -0,0 +1,740 @@
+'use strict';
```

Comment:
We may consider moving it somewhere outside of `internal/crypto` as other WebIDL APIs could use this.

Reply:
See https://github.com/nodejs/node/pull/46067#issuecomment-1370017660